pyglet/image/__init__.py

```diff
@@ -413,6 +442,16 @@ def blit_into(self, source, x, y, z):
         Note that if `source` is larger than this image (or the positioning
         would cause the copy to go out of bounds) then you must pass a
         region of `source` to this method, typically using get_region().
+
+        :Parameters:
```

Comment:
I would remove this section as it's unneeded.
It's also incorrect, as (x, y, z) is the destination, not the anchor point.